src/ContactPeople/ContactPerson/ContactPersonEmails/ContactPersonEmails.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import { Row, Col, KeyValue } from '@folio/stripes/components';
+
+import { EntitiesWithCollapsing } from '../../../common/components';
+import ContactPersonSection from '../ContactPersonSection';
+import ContactPersonItem from '../ContactPersonItem';
+
+const renderEmail = email => (
+  <ContactPersonItem
+    isPrimary={email.isPrimary}
+    key={`${email.value}-${email.description}-${email.language}`}
+  >
+    <Row>
+      <Col xs={3}>
+        <KeyValue label={<FormattedMessage id="ui-organizations.contactPeople.emailAddress" />}>
+          {email.value || '-'}
```

Comment:
I wonder if it was the previous behavior - to show `'-'` in case of empty value. Because in ui-orders it's a common approach to leave a blank space.